apps/services/auth-public-api/src/app/modules/delegations/rsk.config.ts

```diff
@@ -0,0 +1,26 @@
+import { RskModuleConfig } from '@island.is/clients/rsk/v2'
+import { createXRoadAPIPath, XRoadMemberClass } from '@island.is/utils/api'
+import { Base64 } from 'js-base64'
+
+const XROAD_BASE_PATH = process.env.XROAD_BASE_PATH_WITH_ENV ?? ''
+const XROAD_MEMBER_CLASS = XRoadMemberClass.GovernmentInstitution
+const XROAD_MEMBER_CODE = process.env.XROAD_RSK_MEMBER_CODE ?? ''
+const XROAD_API_PATH = process.env.XROAD_RSK_API_PATH ?? ''
+const XROAD_CLIENT_ID = process.env.XROAD_RSK_CLIENT_ID ?? ''
+const USERNAME = process.env.RSK_USERNAME ?? ''
+const PASSWORD = process.env.RSK_PASSWORD ?? ''
+
+export class RskConfig {
+  static get(): RskModuleConfig {
+    return <RskModuleConfig>{
```

Comment:
Better readability to just return a constant object here. No need to create the class with a static method.